aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -370,17 +373,6 @@ inline Return Dispatcher::callWithDispatchKey(const TypedOperatorHandle<Return(A
   return kernel.template call<Return, Args...>(op, std::forward<Args>(args)...);
 }
 
-template<class Return, class... Args>
-inline Return Dispatcher::call(const TypedOperatorHandle<Return(Args...)>& op, Args... args) const {
```

Comment:
There were multiple files that include Dispatcher.h.  I looked through most of them and couldn't find any that called Dispatcher::call() directly. Trying to inline this function mainly because of that + the fact that it's only 7 lines.